starcheck/src/starcheck

```diff
@@ -3,12 +3,12 @@
 unset LD_LIBRARY_PATH
 unset SYBASE
 unset PYTHONPATH
-SKA_bin=`dirname $0`
-eval `${SKA_bin}/flt_envs -shell sh`
+eval `${SKA}/bin/flt_envs -shell sh`
 export PYTHONHOME=$SKA_ARCH_OS
+export SCDIR=${SKA_ARCH_OS}/lib/python2.7/site-packages/starcheck
```

Comment:
To be even more collision safe, you probably don't need to export it.  Also, (and this is getting more invasive), can you just drop the `-sc_data` option and have `starcheck.pl` hardwired to know that its package data root is the same directory where the file itself lives?  This is basically the way that most Python package data works, it does not need to be configurated.